### DISCUSSION TITLE
Matching MacOS Apps

### DISCUSSION THREAD

**Discussion by jdevera** [tfeldmann/organize#408] [Q&A]
I am having trouble matching macos apps. A MacOS app is a folder with a name that ends in `.app`. MacOS treats them specially but at the file system level they are a directory.

I have tried the `extension` filter but I saw in the code that it excludes directories. I also tried with the `name` filter and it did not work, I saw that in the `name` filter the `pathlib.Path`'s `stem` property is used for directories, which 

I cannot find another filter that will give me access to the "extension" part of a directory.

How can I do this?

**Comment by jdevera** [comment]
After trying with other folders, perhaps my issue is that understand the tool wrong, is it not supposed to also match directories?

I tried with a normal directory without dots and I did not get a match either.

**Comment by joev2** [comment]
@jdevera are you still working on this? Did you try using `targets: dirs`? (see [Rule options](https://organize.readthedocs.io/en/latest/rules/#rule-options))
